doc/source/installation.rst

```diff
@@ -2,5 +2,211 @@
+
+.. code-block:: text
+  
+  sudo pacman -Sy openmpi
+
+
+numdiff
+~~~~~~~~
+
+numdiff is used within the automatic testing procedure of Lethe to compare files obtained through floating point arithmetic. Without numdiff, Lethe automatic tests may fail when they should not. Numdiff can be installed directly from your package manager.
+
+.. code-block:: text
+
+  sudo apt-get install numdiff
+
+Regrettably, numdiff is not available in the pacman package manager. It can be downloaded from the following `website <http://www.nongnu.org/numdiff/>`_. If you are using an arch distribution, we assume that you will already know how to carry on with the installation of numdiff. 
+
+P4est
+~~~~~~~
+
+To install p4est, the usual `installation procedure <https://www.dealii.org/current/external-libs/p4est.html>`_ of dealii can be followed.
+
+
+
+Trilinos
+~~~~~~~~~
+
+The installation of Trilinos should be done using the `installation procedure <https://www.dealii.org/current/external-libs/trilinos.html>`_ of dealii:
+
+
+
+METIS
+~~~~~~~
+
+METIS is used for mesh partitioning for parallel computing purposes, specifically in cases with simplex grids. It can be downloaded `here <http://glaros.dtc.umn.edu/gkhome/metis/metis/download>`_ or through candi.
+
+
+
+Installation of dealii
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Clone deal.II from the `official repository <https://github.com/dealii/dealii>`_
+
+.. code-block:: text
+
+  git clone https://github.com/dealii/dealii 
+
+Configure deal.II in a build folder at the same level as the source code
+
+.. code-block:: text
+
+  mkdir build
+  cd build
+
+Depending on how you have installed p4est, Trilinos and METIS, you may need to specify the installation folder of the three libraries
+
+.. code-block:: text
+
+  cmake ../dealii -DDEAL_II_WITH_MPI=ON -DDEAL_II_WITH_TRILINOS=ON -DTRILINOS_DIR=path/to/your/trilinos/installation -DDEAL_II_WITH_P4EST=ON -DP4EST_DIR=path/to/your/p4est/installation  -DDEAL_II_WITH_METIS=ON -DMETIS_DIR=path/to/your/metis/installation -DCMAKE_INSTALL_PREFIX=/path/to/desired/installation`
+
+Compile dealii
+
+.. code-block:: text
+
+  make -j<nprocessor> install
+
+Create an environment variable for the DEALII directory. 
+
+.. code-block:: text
+ 
+  export DEAL_II_DIR=/path/to/dealii/installation
+
+It is generally recommended to add variable to your bashrc so it is always loaded:
+
+.. code-block:: text
+
+ echo "export DEAL_II_DIR=/path/to/dealii/installation" >> ~/.bashrc
+
+.. _install-lethe:
+
+Installation of lethe (Step #2)
+-------------------------------
+
+Clone lethe from the `official repository <https://github.com/lethe-cfd/lethe>`
+
+Create a build folder at the same level as the lethe folder
+
+.. code-block:: text
+
+  mkdir build
+
+Compile Lethe choosing the compilation option (Debug or Release). You can also optionnaly specify and installation direction. We recommend that you do so, since this makes using Lethe much more comfortable.
```

Comment:
```suggestion
Compile Lethe choosing the compilation option (Debug or Release). You can also optionally specify a path to an installation directory of your choice. We recommend that you do so, since this makes using Lethe much more comfortable.
```